src/common/validators.js

```diff
@@ -0,0 +1,44 @@
+import { throwError, splitByDelimiter } from './utils.js';
+import { ERROR_MESSAGE, GAME_RULES } from './constants.js'; 
+
+export const isEmpty = (input) => !input;
```

Comment:
```suggestion
export const isEmpty = (input) => input === "";
```
`isEmpty`라는 이름만 봤을 때는 공백같은 걸 체크하는 것 같은데 실제 로직은 `falsy`한 값은 다 `true`로 리턴하는 것 같네요🤔

지금은 `0`을 인자로 넣어도 `true`로 나올 것 같아요!

함수 이름에 맞는 로직은 `input === ""` 이게 조금 더 가까운 것 같은데 다은님 생각은 어떠신가요??

Reply:
간단하게 작성하려 하다보니, `falsy`한 값이 들어오는 상황을 놓친것 같습니다!
코드 작성하면서 자주 썼던 방식인데, 고쳐야할것 같아요! 예리한 지적 감사합니다!